Include/internal/pycore_codecs.h

```diff
@@ -21,6 +21,17 @@ extern void _PyCodec_Fini(PyInterpreterState *interp);
 
 extern PyObject* _PyCodec_Lookup(const char *encoding);
 
+/*
+ * Un-register the error handling callback function error under the given
+ * name. Only non-standard error handlers can be un-registered.
```

Comment:
rather than "standard" you may say "built-in".

Reply:
Ah thank you, I started with 'native', then moved to 'standard' because of 'standard encodings' but 'built-in' is better!